pkg/cedar/core/client.go

```diff
@@ -0,0 +1,98 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
+
+	"github.com/go-openapi/runtime"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	apiclient "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client"
+	apisystems "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/system"
+	coremodels "github.com/cmsgov/easi-app/pkg/cedar/core/gen/models"
+	"github.com/cmsgov/easi-app/pkg/flags"
+)
+
+const (
+	cedarCoreEnabledKey     = "cedarCoreEnabled"
+	cedarCoreEnabledDefault = false
+)
+
+// NewClient builds the type that holds a connection to the CEDAR Core API
+func NewClient(cedarHost string, cedarAPIKey string, ldClient *ld.LDClient) *Client {
+	fnEmit := func(ctx context.Context) bool {
+		lduser := flags.Principal(ctx)
+		result, err := ldClient.BoolVariation(cedarCoreEnabledKey, lduser, cedarCoreEnabledDefault)
+		if err != nil {
+			appcontext.ZLogger(ctx).Info(
+				"problem evaluating feature flag",
+				zap.Error(err),
+				zap.String("flagName", cedarCoreEnabledKey),
+				zap.Bool("flagDefault", cedarCoreEnabledDefault),
+				zap.Bool("flagResult", result),
+			)
+		}
+		return result
+	}
+
+	hc := http.DefaultClient
+
+	// Uncomment the following if you're having trouble with TLS locally
```

Comment:
Rather than being commenting-based, maybe we could have an environment variable that, if set, uses the commented-out config? (Don't make that change yet, though, I'm going to see if there's a better way to fix the TLS problems when running locally)

Reply:
Removed the commented-out section; it's unnecessary with the CEDAR certs added to the Docker images.